Podcast/Episode.swift

```diff
@@ -310,8 +310,17 @@ class Episode: NSObject, NSCoding {
     }
     
     func createRecommendation(success: ((Bool, Int) -> ())? = nil, failure: ((Bool, Int) -> ())? = nil) {
+        if let user = System.currentUser, let hasRecasted = user.hasRecasted, !hasRecasted { // first recast
```

Comment:
if i'm trying to have this pop up now do I have to make an account? because technically user defaults haven't been written in yet but i've recasted stuff

Reply:
It's specific to the phone and resets on logout .. so every time you log in you would get shown it .. I know @drewsdunne  mentioned that the whole logout reset might get annoying

Reply:
Yeah I would just do once per install. Like once in UserDefaults it's there forever. I feel like I never log out of my apps and if I do it's by accident, and would not want to go through on boarding again, which includes this recasting demo.